src/pages/RocketsCards.js

```diff
@@ -0,0 +1,91 @@
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import {
+  getRocket,
+  reserveRocket,
+  unreserveRocket,
+} from '../redux/rockets/rockets';
+
+const RocketsCards = () => {
+  const dispatch = useDispatch();
+  const fetchApi = 'https://api.spacexdata.com/v3/rockets';
+
+  const myRocketArray = useSelector((state) => state.rockets);
+
+  useEffect(() => {
+    const apiRockets = async () => {
+      const fetchRockets = await fetch(fetchApi);
+      const rockets = await fetchRockets.json();
+      return dispatch(getRocket(rockets));
+    };
+    if (myRocketArray.length === 0) {
+      apiRockets();
+    }
+  }, []);
+
+  const reserveRockets = (e) => {
+    dispatch(reserveRocket(e.target.id));
+  };
+
+  const unReserveRockets = (e) => {
+    dispatch(unreserveRocket(e.target.id));
+  };
+
+  return (
+    <div className="rocket-card">
+      {myRocketArray.map((rocket) => (
+        <div key={rocket.id} className="rockets-div">
+          <div className="rocket-imgs">
+            <img
+              src={rocket.flickr_images}
+              alt={rocket.rocket_name}
+              width="300"
+              height="200"
+              className="header-logo"
+              // eslint-disable-next-line react/jsx-no-duplicate-props
+              className="rocket-img"
+            />
+          </div>
+          <div className="rockets-desc">
+            <h3 className="rockets-desc-title">{rocket.rocket_name}</h3>
+            <p className="rockets-desc-p">
+              {rocket.reserved ? (
+                <button
+                  type="button"
+                  className="reserved-span rocktbtn"
+                  title="button"
+                >
+                  Reserved
+                </button>
+              ) : null}
+              {rocket.description}
+            </p>
+            {rocket.reserved ? (
+              <button
+                type="button"
+                title="button"
+                onClick={unReserveRockets}
+                id={rocket.id}
+                className="unreserve-btn rocktbtn"
+              >
+                Cancel Reservation
+              </button>
+            ) : (
+              <button
+                type="button"
+                title="button"
+                className="rockets-desc-btn rocktbtn"
+                onClick={reserveRockets}
+                id={rocket.id}
+              >
+                Reserve Rocket
+              </button>
+            )}
```

Comment:
_(This is the second iteration of my last comment about the conditional rendering syntax.  Copied it here just for reference.  It's exactly the same requirement as with the rockets)._

- [ ] For this particular project, the React conditional render syntax is required for conditionally displaying both the Rockets and the Missions:

![image](https://user-images.githubusercontent.com/57421823/151388114-547dfc15-0380-44e6-b274-b7f7fbe510bb.png)

Don't worry! I'll show you how!! 💪 💪 

This is an example:

![image](https://user-images.githubusercontent.com/57421823/151388576-018db458-f6d7-4bfc-b0f9-a90dd7768f4f.png)

Another example with a kind of "if/else" scheme:

![image](https://user-images.githubusercontent.com/57421823/151396150-0479bcef-bb3b-44d2-b1a7-e696d222f7d2.png)


The rule is that if the condition on the left of the `&&` is true, the action of the right will be run. 💪 

Now that you know how to do it kindly please implement this syntax to your missions and rockets conditional rendering (i.e. for the "reserved" badge, the "join" badge, etc). ✔️